docs/tutorial/3-tilt-ui.md

```diff
@@ -4,16 +4,14 @@ subtitle: Tilt Tutorial
 layout: docs
 ---
 ## Launching the Web UI
-While your terminal window running `tilt up` is active, if you press `(Spacebar)`, Tilt will open your default browser to the Tilt UI.
-
-If you haven't already done so, go ahead and open it from the terminal now. You can also navigate to [http://localhost:3366]() manually.
+In your terminal window running `tilt up`, press `(Spacebar)`. Tilt will open your default browser to the Tilt UI. (Or you can navigate to [http://localhost:3366]().)
 
 ## Resource Overview
-When you first open the Tilt UI (or click the Tilt logo in the upper left corner), you'll see the resource overview.
+The Resource Overview is the first thing you see when you open the Tilt UI (or click the Tilt logo in the upper left corner).
```

Comment:
Actually, I'm not sure if we follow this convention elsewhere, but I capitalized the names of views here.

Reply:
I think it makes sense!